integration-tests/luks/create-with-ext4.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+# requirements: disk /dev/sdb with empty partition sdb1
+
+
+from sys import exit
+from storage import *
+from storageitu import *
+
+
+set_logger(get_logfile_logger())
+
+environment = Environment(False)
+
+storage = Storage(environment)
+
+staging = storage.get_staging()
+
+sdb1 = Partition.find_by_name(staging, "/dev/sdb1")
+
+luks = sdb1.create_encryption("cr_test")
```

Comment:
While it is true that "encryption" is really a concept, IMHO it is clearly not an activity (that would be the act of encrypting). But since you can perform actions on that thing, it is as much an object as a file system is an object: You can create it, destroy it, set its password. And with the password it also has data which is the other thing that constitutes an object in the OOP sense.

I'd really hate to mess with ugly and artificial sounding things like EncryptionLayer in this context. IMHO this would pretty much destroy the abstraction level we already have with block devices, disks, partition tables, partitions, file systems. I strongly vote for keeping "Encryption" as another one like those.